### PR TITLE
Add Terms and Conditions

### PR DESCRIPTION
**Pull request by maybenikhil**

### CODE REVIEW

File: docs/info/terms-and-conditions.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+party posts on or through Service. However, by posting Content using Service you grant us the right and license to use,
+modify, publicly perform, publicly display, reproduce, and distribute such Content on and through Service. You agree
+that this license includes the right for us to make your Content available to other users of Service, who may also use
+your Content subject to these Terms.
+
+ExpressLRS has the right but not the obligation to monitor and edit all Content provided by users.
+
+In addition, Content found on or through this Service are the property of ExpressLRS or used with permission. You may
+not distribute, modify, transmit, reuse, download, repost, copy, or use said Content, whether in whole or in part, for
+commercial purposes or for personal gain, without express advance written permission from us.
```
Comment (deadbytefpv) [inline docs/info/terms-and-conditions.md:67]: "without express advance written permission from us."
I don't think this applies, so might as well put in "without sufficient attribution to the source material."

**Review by deadbytefpv — CHANGES_REQUESTED** [review]
I understand that the content came from a third-party service so there could be inaccuracies.

I would urge the others in the Dev team to do their review as well.
Even though not many do read these pages, we should ensure this is accurate for those who do.

File: docs/info/terms-and-conditions.md [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+may send. However, you may opt out of receiving any, or all, of these communications from us by following the
+unsubscribe link or by emailing at info@expresslrs.org.
+
+## 3. Contests, Sweepstakes and Promotions
+
+Any contests, sweepstakes or other promotions (collectively, “Promotions”) made available through Service may be
+governed by rules that are separate from these Terms of Service. If you participate in any Promotions, please review the
+applicable rules as well as our Privacy Policy. If the rules for a Promotion conflict with these Terms of Service,
+Promotion rules will apply.
+
```
[the remark this answers is not in the document]
Comment (maybenikhil) [reply docs/info/terms-and-conditions.md:43]: may as well keep it, highly unlikely but no issue with keeping it imo

**Comment by maybenikhil** [comment]
i feel that saying "Service" not "the Service" lines up with legal speak and how Service is defined in the first section

**Review by deadbytefpv — APPROVED** [review]

**Pull request merged**